bookbotproject/telegrambot.py

```diff
@@ -0,0 +1,51 @@
+import json
+import requests
+import time
+
+TOKEN = "433004356:AAFzeqBEW8_UgEPDOnJ8bnQAPitaR7gLSSo";
+URL = 	"https://api.telegram.org/bot{}/".format(TOKEN);
+
+def send_request(url):
+	res = requests.get(url);
+	content = res.content.decode("utf8");
+	return content;
+
+
+def get_json(url):
+	content = send_request(url);
+	retrieved_json = json.loads(content);
+	return retrieved_json;
+
+
+def get_updates(offset=None):
+	updateURL = URL + "getUpdates?timeout=100"
+	if offset:
+		updateURL += "&offset={}".format(offset);
+	_json = get_json(updateURL);
+	return _json;
+
+def get_last_chat(updates):
+	length = len(updates["result"]);
+	text = updates["result"][length-1]["message"]["text"];
+	chat_id = updates["result"][length-1]["message"]["chat"]["id"];
+	update_id = updates["result"][length-1]["update_id"]; 
+	return [text, chat_id, update_id];
+
+def send_message(chat_id):
+	sendURL = URL + "sendMessage?text={}&chat_id={}".format("jamiryo",chat_id);
+	send_request(sendURL);
+
+def main():
+	last_chat = (None, None);
+	last_update = None;
+	while True:
+		text, chat, update_id = get_last_chat(get_updates(last_update));
+		if(text, chat) != last_chat:
+			send_message(chat);
```

Comment:
Since we are updating the send_message function signature, we need to update here too.
I have something like this in mind: send_message(_message_, chat_id)
Where _message_ is calculated in a separate function like getNextMessageByResponse().
This function should take in the response from user (the "text" variable above), and the chat_id. It should return the response as a string.

Currently we don't have the requirements to fill that function but lets create the function first and put a _TODO: will calculate the next message after wit.ai is ready_ inside.

You can return "jamiryo" by default there. :)